src/pyrenew/process/differencedprocess.py

```diff
@@ -198,15 +193,23 @@ def sample(
             Whose value entry is a single array representing the
             undifferenced timeseries
         """
-        diffs, *_ = self.fundamental_process.sample(
-            *args,
-            n=(n - self.differencing_order),
-            init_vals=fundamental_process_init_vals,
-            **kwargs,
-        )
+        if n < 1:
```

Comment:
also check for integer?

Reply:
addressed in 755fc39

My only question is whether we'd like to allow `n=jnp.array(5)`, but I think we can make this less strict if and when that becomes a probelm.